man/systemd-networkd-wait-online.service.xml

```diff
@@ -85,6 +85,30 @@
         </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>-4</option></term>
+        <term><option>--ipv4</option></term>
+
+        <listitem><para>Waiting for an IPv4 address of each network interface in addition to the range of operational
+        states. If the argument <option>--any</option> will be passed, then <command>systemd-networkd-wait-online</command>
+        will wait for one IPv4 address at least. The option is applied only for the operational state <literal>degraded</literal>
+        or exceeded. If the address family is not specified here, then the value from <varname>RequiredFamilyForOnline=</varname>
```

Comment:
`exceeded` -> `above`?

`If the address family is not specified here,` -> `If neither <option>--ipv4</option> nor <option>--ipv6</option> is specified,`